### ISSUE TITLE
Pythone.exe crush and not working when run python3 landmark.py

### ISSUE THREAD

**Issue by abdou31** [yinguobing/cnn-facial-landmark#36]
Hello Sir ,
+ Cuda 10.0
+ Cudnn 7.5
+ Graphic Card : Nvidia Geforce 840m 
+ Windows 10 x64
+ Tensorflow 1.13
+ OpenCV 4.1
+ Python 3.6
When i run this command : ( I don't if this command do training or no )  

>  python3 landmark.py

i get this error : 
![image](https://user-images.githubusercontent.com/19480228/57513656-11180000-730f-11e9-8ce5-0667a53744b0.png)

When i debug the project with Visual Studio 2017 : 

![image](https://user-images.githubusercontent.com/19480228/57513831-7835b480-730f-11e9-93b8-5dfb75258a99.png)

You say in the readme that tensorflow 1.13 may causes problem , and i should update the code , What kind of change did should i do in the source code landmark.py? 
Thanks,

**Comment by yinguobing** [comment]
I'm not familiar with Visual Studio, and it's hard to tell the reason that caused this problem.

Try following the official TensorFlow tutorial to make sure your installation is valid first.

**Comment by abdou31** [comment]
I follow the tensorflow installation and it work, i think that i find the source of the problem , i don't have any idea about how to train or predict or valid model and i dont have train.record , test.record and valid.record 
I download test.record file and when i test the error doesn't show again but i get other problem in cv:: function
I will post the problem that i get 
I followed for now someone who explain the train and how to organize data ( i have only 300WV dataset , i can't use 300W dataset beacause the archive is corrupted after dowload it

**Comment by yinguobing** [comment]
The 300W dataset is not corrupted, it's compressed and split in multiple parts. Follow the official instruction to unzip it.

**Comment by abdou31** [comment]
How can i unzip it? did you have any idea? and what about training ? why you don't have tutorial explain the process of training and testing to follow it ?

**Comment by yinguobing** [comment]
No tutorial is perfect, there will always be issues you need to solve by yourself.
For TensorFlow beginners the official tutorial should be the first step to understand how this deep learning framework works. This tutorial is a little difficult as the problem to be solved involves the whole process from data downloading to model distributing. I tried my best to keep the tutorial easy to understand by it's still hard to keep every details in record.

If you find it hard to follow, you might want to pause a while and try to figure out which part is missing.

**Comment by abdou31** [comment]
I follow this tutorial for now :  
+ https://github.com/faust690226/cnn-facial-landmark-tutorial

I think is good to start training 300W data , if everything works , i will start training my own dataset ( iris ) 
What about training own dataset is it the same process?
